src/main/java/com/gooddata/md/MetadataService.java

```diff
@@ -97,6 +101,30 @@ public MetadataService(RestTemplate restTemplate) {
     }
 
     /**
+     * Retrieves a list of objects corresponding to the supplied list of URIs.
+     *
+     * @param project project that contains the objects to be retrieved
+     * @param uris list of URIs
+     * @return list of metadata objects corresponding to the supplied URIs
+     */
+    public List<Obj> getObjsByUris(Project project, List<String> uris) {
```

Comment:
@martiner do we really need `byUris`? The uri is in fact redundant information - project and obj id should be enough? Would be nice to see the usecase

Reply:
well, typically you retrieve object uris from other requests (query resource, or various elements of content that contain just the uri) and then you want to retrieve the whole objects for all this uris in bulk. So if the client had to pass in object IDs, it would actually be more work for the client (would have to translate the URIs from the payload of the other requests to object ids).